Plan/common/src/main/resources/assets/plan/web/network.html

```diff
@@ -557,32 +567,49 @@ <h6 class="m-0 font-weight-bold col-black"><i
+                                            Insights for 30 days</h6>
+                                    </div>
+                                    <div class="card-body" id="data_insights">
+                                        <p><i class="fa fa-fw fa-user col-light-green"></i> New <i
+                                                class="fa fa-fw fa-long-arrow-alt-right"></i> <i
+                                                class="fa fa-fw fa-user col-lime"></i> Regular<span
+                                                class="float-right"><b><span
+                                                id="data_new_to_regular"></span></b></span></p>
+                                        <p><i class="fa fa-fw fa-user col-lime"></i> Regular <i
+                                                class="fa fa-fw fa-long-arrow-alt-right"></i> <i
+                                                class="fa fa-fw fa-user col-blue-grey"></i> Inactive<span
+                                                class="float-right"><b><span
+                                                id="data_regular_to_inactive"></span></b></span></p>
+                                        <p class="float-right"><i class="text-success fa fa-caret-up"></i><i
+                                                class="text-danger fa fa-caret-down"></i>
+                                            <small>Comparing 30d ago to Now</small>
+                                        </p>
+                                    </div>
                                 </div>
-                                <div class="card-body" id="data_insights">
-                                    <p><i class="fa fa-fw fa-user col-light-green"></i> New <i
-                                            class="fa fa-fw fa-long-arrow-alt-right"></i> <i
-                                            class="fa fa-fw fa-user col-lime"></i> Regular<span
-                                            class="float-right"><b><span
-                                            id="data_new_to_regular"></span></b></span></p>
-                                    <p><i class="fa fa-fw fa-user col-lime"></i> Regular <i
-                                            class="fa fa-fw fa-long-arrow-alt-right"></i> <i
-                                            class="fa fa-fw fa-user col-blue-grey"></i> Inactive<span
-                                            class="float-right"><b><span
-                                            id="data_regular_to_inactive"></span></b></span></p>
-                                    <p class="float-right"><i class="text-success fa fa-caret-up"></i><i
-                                            class="text-danger fa fa-caret-down"></i>
-                                        <small>Comparing 30d ago to Now</small>
-                                    </p>
+                            </div>
```

Comment:
It is best not to add a nested column here (might lead to some issues on smaller screens), instead adding the new pie as a card
```suggestion
```